ocs_ci/helpers/helpers.py

```diff
@@ -1729,6 +1729,57 @@ def verify_volume_deleted_in_backend(
         return False
 
 
+def delete_volume_in_backend(interface, img_uuid, pool_name=None):
+
+    """
+    Delete an Image/Subvolume in the backend
+
+    Args:
+         interface (str): The interface back the pvc (rbd, cephfs)
+            For now only support RBD. Cephfs support will be added
+            later.
+         img_uuid (str): Part of VolID which represents corresponding
+            image/subvolume in backend, eg:
+            ``oc get pv/<volumeName> -o jsonpath='{.spec.csi.volumeHandle}'``
+            Output is the CSI generated VolID and looks like:
+            ``0001-000c-rook-cluster-0000000000000001-f301898c-a192-11e9-852a-1eeeb6975c91``
+            where image_uuid is ``f301898c-a192-11e9-852a-1eeeb6975c91``
+         pool_name (str): The of the pool incase of RBD.
```

Comment:
- `The name of the pool...`
- Why only RBD?

Reply:
Because cephfs has only one defalut pool and doesn't support multiple storageclass.

Reply:
so what if CephFS pool is passed to this function? shouldn't we have a condition for that?

Reply:
Full support for cephfs and rbd.